polynote-kernel/src/main/scala/polynote/kernel/interpreter/python/PythonInterpreter.scala

```diff
@@ -166,9 +166,9 @@ class PythonInterpreter private[python] (
   }
 
   def init(state: State): RIO[InterpreterEnv, State] = for {
+    globals <- getValue("globals().copy()")
```

Comment:
@jeremyrsmith this also includes this fix to make sure our predef stuff isn't in globals.